front/src/components/AddMockupSurvey.js

```diff
@@ -339,6 +368,13 @@ class AddMockupSurvey extends Component {
                                                                     className="btn btn-primary btn-margin-top">Add Statement
                                                                 </Button>
                                                             </Col>
+                                                            <Col>
```

Comment:
actually creating this column could be extracted to a function. e.g. (maybe with better function name):
```
    getHighlightCol(index) {
        return <Col>
            Highlight doesn't work with scrolling mockups
            <Button
                onClick={(event) => this.addHighlight(event, index)}
                className="btn btn-primary btn-margin-top">Add Highlight
            </Button>
        </Col>;
    }
```